### ISSUE TITLE
Nitpicking about demoparties vs LAN parties

### ISSUE THREAD

**Issue by Kabuto** [pengan1987/intro-to-demoscene#2]
Regarding chapter 10 and the snippet "that does not include game content":

Most demoparties do allow for homebrew games to be entered in competitions.

Some have explicit game competitions (such as Revision) while those without a game compo typically allow for games to be entered in the Wild category.

Was the intended meaning something like "that discourage gaming"? Quote from 2019's Revision party:

> No games at the party.
> This is a demoscene party, dedicated to computer art, not a LAN or Leecher event. Gamers will be thrown out!

**Comment by pengan1987** [comment]
Yes, your understanding is correct, I saw the “no games” rule on Revision as early as 2013
https://2013.revision-party.net/about/rules
I think it should state it differently
